lib/browser/api/menu.js

```diff
@@ -125,18 +125,20 @@ Menu.prototype._init = function () {
       var checked, group, id, j, len, radioItem, ref1
       ref1 = this.groupsMap
       for (id in ref1) {
-        group = ref1[id]
-        checked = false
-        for (j = 0, len = group.length; j < len; j++) {
-          radioItem = group[j]
-          if (!radioItem.checked) {
-            continue
+        if(ref.hasOwnProperty(id))
```

Comment:
@poiru Not sure how much of a concern this is for us but performance of `for ... of` loops are still quite bad in Chromium.  Like easily twice as slow as a standard `for (let i, i < thing.length; i++)` loop.